Cargo.lock

```diff
@@ -103,9 +103,9 @@ dependencies = [
 
 [[package]]
 name = "cc"
-version = "1.0.72"
+version = "1.0.78"
```

Comment:
Can we update only the strictly required ones ?

Reply:
Ok... but it's quite hard and need to change `Cargo.lock`  by hand ... I wold like avoid to do it

Reply:
Rollback to the previous version and replace crossbeans-channels to 0.5.1 that is not yanked and closer to 0.5.2 one